lab-katy/__test__/image-router.test.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const request = require('superagent');
+const server = require('../server.js');
+const serverToggle = require('../lib/server-toggle.js');
+
+const Image = require('../model/image.js');
+const User = require('../model/user.js');
+const Collection = require('../model/collection.js');
+
+require('jest');
+
+const url = 'http://localhost:3001';
```

Comment:
This is breaking on my computer because everywhere else you have `process.env.PORT || 3001`, and so the server is starting on port 3000 for me, but then this tries to run on 3001 and crashes out. This needs to be fixed.